maintainers/working_groups/learning_journey/documentation-survey.md

```diff
@@ -137,6 +137,12 @@ To better navigate the material and judge its relevance, every entry should prov
 
 ### Nixpkgs
 - https://nixos.org/guides/nix-pills/index.html
+  
+   Index to sections: Why try it; Installing; Environment; Language; Functions & Imports; Derivations; Generic Builders, Dependencies; Nix Shell; Garbage Collection; Design Patterns; Search Paths; NixPkgs Parameters & Overriding; Store Paths; Stdenv; Dependencies & Hooks.
+  - Reading time: 5 min
+  - Author: Luca Bruno
+  - Created: 2014 Updated 2017
+    
```

Comment:
```suggestion  
```

This is an auto-generated table of contents. Not sure how it got in here to begin with. Let's just remove it.